tests/test_import.py

```diff
@@ -1,13 +1,29 @@
 import pytest
+from llama_parse.utils import Language
 
 from megaparse.core.megaparse import MegaParse
 from megaparse.core.parser.unstructured_parser import UnstructuredParser
+from megaparse.sdk.megaparse_sdk import MegaParseSDK
+from megaparse.sdk.megaparse_sdk.utils.type import StrategyEnum
 
 
 @pytest.mark.skip("slow test")
 def test_load():
     parser = UnstructuredParser(model=None)
     megaparse = MegaParse(parser)
-    response = megaparse.load("./tests/data/files/dummy.pdf")
+    response = megaparse.load("./tests/data/dummy.pdf")
     print(response)
-    assert response.strip("\n") == "# Dummy PDF download"
+    assert response.strip("\n") == "Dummy PDF download"
+
+
+# @pytest.mark.skip("slow test")
+@pytest.mark.asyncio
+async def test_load_SDK():
```

Comment:
Is this a unit test? an integration test ? a performance test?

Reply:
Integration test ? 
What should I modify ?